### DISCUSSION TITLE
Dokumentation für Integration mit UV

### DISCUSSION THREAD

**Discussion by joWakaT** [urlaubsverwaltung/zeiterfassung#372] [Q&A]
Was muss konfiguriert werden um UV und Zeiterfassung miteinander zu integrieren?

https://github.com/urlaubsverwaltung/zeiterfassung/pull/290

- Krankmeldungen aus der UV abfragen und anzeigen https://github.com/urlaubsverwaltung/zeiterfassung/issues/297
- Urlaubstage aus der UV abfragen und anzeigen https://github.com/urlaubsverwaltung/zeiterfassung/issues/3

**Comment by grafjo** [comment]
Hi @joWakaT - diese Integration ist aktuell nur über das Angebot von https://urlaubsverwaltung.cloud/ verfügbar.

**Comment by reneneee** [comment]
Hi @grafjo ,
Ist es geplant, diese Integration zukünftig auch in der selbst gehosteten Variante zur Verfügung zu stellen?

**Reply by grafjo** [reply]
Hi @reneneee,
es ist durchaus möglich diese Integration auch On-Premise zu verwenden, jedoch wird diese Erweiterung u.a. Lizenzkosten verursachen. Falls das für dich interessant ist, schreib bitte eine Mail an info@urlaubsverwaltung.cloud.

**Reply by ryrun** [reply]
@grafjo Hätten da ggf. auch Interesse. Testen gerade beide Anwendungen intern. Würde mich bei Bedarf melden.
